docs/releases/7.0.md

```diff
@@ -81,6 +81,59 @@ This version adds formal support for Django 5.2.
  * Migrate setuptools configuration from `setup.py` and `setup.cfg` to `pyproject.toml` (Sage Abdullah)
  * Refactor `move_choose_destination` to a class-based view (Chiemezuo Akujobi)
 
+## Upgrade considerations - removal of deprecated features from Wagtail 5.2 - 6.3
+
+Features previously deprecated in Wagtail 5.2, 6.0, 6.1, 6.2 and 6.3 have been fully removed. For additional details on these changes, see:
+
+ * [Wagtail 5.2 release notes](/releases/5.2)
+ * [Wagtail 6.0 release notes](/releases/6.0)
+ * [Wagtail 6.1 release notes](/releases/6.1)
+ * [Wagtail 6.2 release notes](/releases/6.2)
+ * [Wagtail 6.3 release notes](/releases/6.3)
+
+The most significant changes are highlighted below.
+
+### Removal of `classnames` attribute on menu item and image format classes
+
+The `classnames` keyword argument on the following classes is no longer supported and should be replaced with `classname`:
+
+ * `admin.menu.MenuItem`
+ * `admin.ui.sidebar.ActionMenuItem`
+ * `admin.ui.sidebar.LinkMenuItem`
+ * `admin.ui.sidebar.PageExplorerMenuItem`
+ * `contrib.settings.registry.SettingMenuItem`
+ * `wagtail.images.formats.Format`
+
+### Renamed settings
+
+ * The `WAGTAIL_AUTO_UPDATE_PREVIEW` setting is removed and should be replaced with `WAGTAIL_AUTO_UPDATE_PREVIEW_INTERVAL = 0` to disable auto-update.
+ * The `PASSWORD_REQUIRED_TEMPLATE` setting is no longer recognised and should be replaced with `WAGTAIL_PASSWORD_REQUIRED_TEMPLATE`.
+ * The `DOCUMENT_PASSWORD_REQUIRED_TEMPLATE` setting is no longer recognised and should be replaced with `WAGTAILDOCS_PASSWORD_REQUIRED_TEMPLATE`.
```

Comment:
The password template settings don't have individual targets (and not worth adding IMO), but linking to the main "Frontend authentication" heading should be close enough.

```suggestion
 * The `WAGTAIL_AUTO_UPDATE_PREVIEW` setting is removed and should be replaced with [`WAGTAIL_AUTO_UPDATE_PREVIEW_INTERVAL = 0`](wagtail_auto_update_preview_interval) to disable auto-update.
 * The `PASSWORD_REQUIRED_TEMPLATE` setting is no longer recognised and should be replaced with [`WAGTAIL_PASSWORD_REQUIRED_TEMPLATE`](frontend_authentication).
 * The `DOCUMENT_PASSWORD_REQUIRED_TEMPLATE` setting is no longer recognised and should be replaced with [`WAGTAILDOCS_PASSWORD_REQUIRED_TEMPLATE`](frontend_authentication).
```